practices/practice-one/src/helpers/firebase/config.ts

```diff
@@ -0,0 +1,21 @@
+import { initializeApp } from "firebase/app";
+import { getAnalytics } from "firebase/analytics";
+import { getAuth } from 'firebase/auth'
+import { getDatabase } from 'firebase/database'
+
+const firebaseConfig = {
+    apiKey: "AIzaSyDujQ211O3uvRSZFfzG27NG9g9H0mKvfJg",
```

Comment:
I mean config information such as: apiKey. It should be placed in env file

Reply:
I put it all in the .env file